### PR TITLE
Update dependencies and skip symlink test on windows

### PR DESCRIPTION
**Pull request by mceachen**
This PR fixes the "outdated dependencies" warning badge and should let you run tests on appveyor if you want.

Thanks for making `cp-cli`!

### CODE REVIEW

File: test/cp-cli.test.ts [hunk 1]
```diff
@@ -29,16 +30,18 @@ describe('cp-cli', () => {
...
-    shell.cd('../..');
-    const { stderr } = shell.exec('node bin/cp-cli -d test/assets out');
-    expect(stderr).toEqual('');
-    let stats = await fse.stat('out/foo.txt');
-    expect(stats.isFile()).toEqual(true);
-    stats = await fse.stat('out/bar.txt');
-    expect(stats.isFile()).toEqual(true);
-    expect(stats.isSymbolicLink()).toEqual(false);
-  });
+  if (platform() !== 'win32') {
```
Comment (mceachen) [inline test/cp-cli.test.ts]: It's too bad jest doesn't have `mocha`'s `skip()`.

Comment (screendriver) [reply test/cp-cli.test.ts]: You [can](https://facebook.github.io/jest/docs/en/api.html#testskipname-fn) 😉

**Comment by screendriver** [comment]
Because I added some stuff I also did an upgrade to all dependencies and copied your if statement for Windows.

Thank you for your help! 👍

File: test/cp-cli.test.ts (hunk 1 above)
Comment (mceachen) [reply test/cp-cli.test.ts]: Jest's `.skip` only supports static skipping. If you look at `mocha's` (.skip)[https://mochajs.org/#inclusive-tests] it lets you conditionally call it. The way this code is now, the test is silently omitted on windows (which isn't ideal).

Comment (screendriver) [reply test/cp-cli.test.ts]: Ah I understand. That's a cool feature and I didn't knew that. Maybe [skip-if](https://github.com/relekang/skip-if) is what you are looking for :wink: Unfortunately it's not build in into Jest: https://github.com/facebook/jest/issues/3652

Comment (mceachen) [reply test/cp-cli.test.ts]: Yeah, I didn't want to add a dependency to your module just for this. Thanks for releasing a new version!

**Pull request closed without merging**